cmake/match.cmake

```diff
@@ -6,6 +6,8 @@
 # with a known good match file.
 #
 
+find_package(Python3 COMPONENTS Interpreter)
```

Comment:
Yes, because this is a separate instance of the cmake process.
```
add_test(NAME ${TEST_NAME}
    COMMAND ${CMAKE_COMMAND} -P ${PROJECT_SOURCE_DIR}/cmake/match.cmake
    ...
)
```
This starts a new cmake process to run the provided script. It's unfortunate, but that's how it works. I could replace this entire `match.cmake` script with a python one, which would simplify things (but that's a separate PR).